docs/Apps/Metabolomic Data/Untargeted Pipeline.md

```diff
@@ -360,66 +364,108 @@ Once the parameters are specified, click on the *Update* button to plot the volc
 
 ![Volcano plot](../../img/UntargetedPipeline/utp_app_volcano_plot.png) <center>**Figure 48.** Volcano plot</center>
 
-*Filtered Metabolites Visualization* provides the visualization of cohort-based distribution of the metabolites that are significant based on the parameters specified.
+*Filtered Metabolites Visualization* provides the option to visualize the cohort-based distribution of the features/metabolites that are significant based on the parameters specified through bar plots and boxplots and also look at the EIC plot of specific features to update categorization of the feateure by the peakML algorithm.
 
 ![Filtered Metabolites Visualization](../../img/UntargetedPipeline/utp_app_stats_analysis_filtered_metabolites_visualization.png) <center>**Figure 49.** Filtered Metabolites Visualization</center>
 
+*   Plots: This tab allows the user to visualize the cohort-based distribution of the features/metabolites through barplots and boxplots.
+![Filtered Metabolites Plots](../../img/UntargetedPipeline/utp_app_stats_analysis_filtered_metabolites_barplot_boxplot_visualization.png) <center>**Figure 50.** Barplot </center>
+*   Heatmap: This tab allows you to produce a heatmap of the filtered features/metabolite data, so that you can observe the level of expression in a visual form. Click on *Load Heatmap* button to generate the heatmap..
+![Heatmap of the significant features/metabolites data](../../img/UntargetedPipeline/utp_app_filtered_heatmap.png) <center>**Figure 51.** Heatmap</center>
+*   EIC Plot: This tab allows you to visualize the eic of specific features so that the user can relabel the classification of the feature done by the peakML algorithm. You can also download the updated emdb file after reaclassification from the Downloads tab.
```

Comment:
1. EIC in capitals
2. So that you * can relabel the classification of the features*
3. reclassification*
4. Downloads tab in italics